dashboard/app/reporting.go

```diff
@@ -227,9 +228,43 @@ func handleReportNotif(c context.Context, typ string, bug *Bug) (*dashapi.BugNot
 		commits := strings.Join(bug.Commits, "\n")
 		return createNotification(c, dashapi.BugNotifBadCommit, true, commits, bug, reporting, bugReporting)
 	}
+	for _, label := range bug.Labels {
```

Comment:
I think this should go before "bad fix commit". These can persist for months.

Reply:
They shouldn't interfere much. In this function we generate a message to be sent right now and e.g. for `bad fix commit` we rely on `LastActivity` in order not to send it every minute.